.github/workflows/go.yml

```diff
@@ -147,6 +147,31 @@ jobs:
           version: ${{ env.KIND_VERSION }}
           skipClusterCreation: true
 
+#      - name: Set up Helm
+#        uses: azure/setup-helm@v1
+#        with:
+#          version: v3.4.0
+
+      - name: Setup Kind Cluster
+        run: |
+          kind delete cluster
+          kind create cluster --image kindest/node:v1.18.15
+          kubectl version
+          kubectl cluster-info
+
+      - name: helm lint
+        run: |
+          helm lint ./charts/vela-core
+
+      - name: helm install
+        run: |
+          helm install --create-namespace -n vela-system kubevela ./charts/vela-core/ --timeout 15m
+
+      - name: helm test
+        run: |
+          helm test -n vela-system kubevela --timeout 5m
+          kubectl -n vela-system logs kubevela-application-test
```

Comment:
please check the `make e2e-setup` command, in the makefile, it has already install the helm chart. We can just merge `helm lint` and `helm test` into the `e2e-setup`.

Reply:
OK, I will take a look.